src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+li,
+input,
+button,
+textarea {
+  padding: 0;
   margin: 0;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__item {
+  list-style: none;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  /* margin: 0 10px; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  height: 60px;
```

Comment:
Both `height` and `line-height` are not needed here. One of them would be fine, just make sure that the text is vertically centered.